### ISSUE TITLE
Zoom does not work with zoom.profile; stuck at "connecting"

### ISSUE THREAD

**Issue by Privacy-Badger-User** [netblue30/firejail#3428]
Zoom's Linux app will open with the standard profile in /etc/firejail, and will let me log in to zoom, but is thereafter non-functional.  "Connecting ..." shows at the top of the app screen.  I am unable to access or start the Zoom meetings I host.  The Linux zoom app works fine if run outside of firejail.

Firejail --version: 0.9.62, installed from the openSUSE Tumbleweed repositories

openSUSE Tumbleweed 20200519

I have attached the output of **firejail --debug zoom** and the existing firejail zoom profile in /etc/firejail.  I apologize if I have omitted anything.  I rarely use github.  

[firejail_debug.txt](https://github.com/netblue30/firejail/files/4666993/firejail_debug.txt)

[firejail.profile.txt](https://github.com/netblue30/firejail/files/4666997/firejail.profile.txt)

**Comment by glitsj16** [comment]
Might be #3272, which is fixed in master but not in 0.9.62. You can try adding the below to a zoom.local override file:

```
protocol unix,inet,inet6,netlink
ignore seccomp
seccomp !chroot
```

**Comment by Privacy-Badger-User** [comment]
Fixed!  I created the zoom.local file in ~/.config/firejail, tried again.  Problem solved.  Thank you!!!
